### PR TITLE
Globals UI should limit creation of globals to org limit

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://github.com/rapidpro/rapidpro/issues/1360

### CODE REVIEW

File: temba/globals/tests.py [hunk 1]
```diff
@@ -104,7 +105,7 @@ def test_list_views(self):
 
         self.assertListFetch(unused_url, allow_viewers=False, allow_editors=False, context_objects=[self.global2])
 
-    @override_settings(MAX_ACTIVE_GLOBALS_PER_ORG=4)
```
Comment (rowanseymour) [inline temba/globals/tests.py:107]: Oh weird.. so we have both this setting and the constant in `Global`.. and they're different numbers.. my bad 🤦 

And I see we have `ContactGroup.MAX_ORG_CONTACTGROUPS` but `settings.MAX_ACTIVE_CONTACTFIELDS_PER_ORG`

I think eventually these become settings on org (@nicpottier ?).. but in the name of consistency let's go with these settings:

```
MAX_ACTIVE_CONTACTFIELDS_PER_ORG = 250
MAX_ACTIVE_CONTACTGROUPS_PER_ORG = 250
MAX_ACTIVE_GLOBALS_PER_ORG = 250
```

For bonus points you could make it a method of org like..

```
org.get_limit(Org.LIMIT_FIELDS)
org.get_limit(Org.LIMIT_GLOBALS)
org.get_limit(Org.LIMIT_GROUPS)
```

That for now just returns those settings values but in future reads from something on the org model.

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/globals/tests.py (hunk 1 above)
Comment (norkans7) [reply temba/globals/tests.py:107]: Updated the PR

File: temba/settings_common.py [hunk 2]
```diff
@@ -1219,7 +1219,3 @@
 
 # ElasticSearch configuration (URL RFC-1738)
 ELASTICSEARCH_URL = os.environ.get("ELASTICSEARCH_URL", "http://localhost:9200")
-
-# Maximum active objects are org can have
```
Comment (norkans7) [inline temba/settings_common.py:1223]: We lost the ability to customize these, I am not sure if anyone was using that

**Review by rowanseymour — APPROVED** [review]
👍

**Pull request merged**